addons/nvidia-bootstrap/image/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# Simple early detection of nvidia card
+grep 10de102d /proc/bus/pci/devices || exit 0
+
+# p2.xlarge
+# 00f0	10de102d	4b	        84000000	      100000000c	               0	        8200000c	               0	               0	               0	         1000000	       400000000	               0	         2000000	               0	               0	               0	nvidia
+
+
+# This is pretty annoying.... note this is installed onto the host
+chroot /rootfs apt-get update
+chroot /rootfs apt-get install --yes gcc
+
+mkdir -p /rootfs/tmp
+cd /rootfs/tmp
+# TODO: We can't download over SSL - presents an akamai cert
+wget http://us.download.nvidia.com/XFree86/Linux-x86_64/375.39/NVIDIA-Linux-x86_64-375.39.run
```

Comment:
personally I'd rather prefer to have https with cert mismatch ignored, vs not using https at all just because of the CDN cert